### PR TITLE
Fixed export for optimized build

### PR DESCRIPTION
**Pull request by KevinVanH**
# Description

Please include a summary of the change, which issue is fixed and the context if the new feature. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant and add new options if necessary. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: management/src/services/newsArticle.js [hunk 1]
```diff
@@ -47,4 +47,5 @@ const deleteNewsArticle = async (id) => {
     });
 };
 
-module.exports = { getNewsArticles, createNewsArticle, updateNewsArticle, deleteNewsArticle };
\ No newline at end of file
+//module.exports = { getNewsArticles, createNewsArticle, updateNewsArticle, deleteNewsArticle };
```
Comment (FedorTselishchev) [inline management/src/services/newsArticle.js:50]: even commets er uit halen

File: management/src/services/userImage.js [hunk 2]
```diff
@@ -41,4 +41,5 @@ const handleUserImage = async (body, action) => {
     });
 };
 
-module.exports = { getUserImages, approveUserImage, declineUserImage, restoreUserImage, deleteUserImage };
\ No newline at end of file
+// module.exports = { getUserImages, approveUserImage, declineUserImage, restoreUserImage, deleteUserImage };
```
Comment (FedorTselishchev) [inline management/src/services/userImage.js:44]: even commets er uit halen

**Review by FedorTselishchev — APPROVED** [review]
vlg goed, alleen comments nog er uit yeeten

**Review by BjornVanBruchem — APPROVED** [review]
Comments verwerken

**Pull request merged**